docs/_sass/hudi_style/_variables.scss

```diff
@@ -54,7 +54,7 @@ $light-gray: mix(#fff, $gray, 50%) !default;
 $lighter-gray: mix(#fff, $gray, 90%) !default;
 
 $background-color: #fff !default;
-$code-background-color: #fafafa !default;
+$code-background-color: #f3f3f3 !default;
```

Comment:
This line changed on purpose?

Reply:
Darken background color.

![image](https://user-images.githubusercontent.com/20113411/73608913-484be100-4603-11ea-8aed-7cb2cbf88e22.png)